src/index.ts

```diff
@@ -12,17 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-// ** This file is automatically generated by gapic-generator-typescript. **
-// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** This file is automatically generated by synthtool. **
+// ** https://github.com/googleapis/synthtool **
 // ** All changes to this file may be overwritten. **
 
-import * as v1 from './v1';
 import * as v1beta1 from './v1beta1';
+import * as v1 from './v1';
+
 const ContainerAnalysisClient = v1.ContainerAnalysisClient;
-const GrafeasClient = v1beta1.GrafeasV1Beta1Client;
-export {v1, v1beta1, ContainerAnalysisClient, GrafeasClient};
-// For compatibility with JavaScript libraries we need to provide this default export:
-// tslint:disable-next-line no-default-export
-export default {v1, v1beta1, ContainerAnalysisClient, GrafeasClient};
+
```

Comment:
```
const GrafeasClient = v1beta1.GrafeasV1Beta1Client;
```
This is a breaking change, not sure if we still want to expose this client to users. For my understanding containerAnalyisis only `reroute` to use this interface and grafeas should be not visible to users. WDYT? @alexander-fenster

Reply:
Oh, I totally missed it. We don't want to make a breaking change here, let's put it back please (maybe with a synth hack). @summer-ji-eng 

I'll remove `automerge` for now, thank you @xiaozhenliu-gg5 for noticing it!

Reply:
Thanks for reminding.

I will hold this PR for later and count as a category that export beta client library, like video-intelligence. And discuss with team how to deal with these libraries.